lib/src/render/image/image_upload_widget.dart

```diff
@@ -84,8 +93,47 @@ class _ImageUploadMenuState extends State<ImageUploadMenu> {
     super.dispose();
   }
 
+  void _pickFiles() async {
+    _resetState();
+    try {
+      _paths = (await FilePicker.platform.pickFiles(
+              type: FileType.image,
+              onFileLoading: (FilePickerStatus status) =>
+                  _logException(status.toString()),
+              allowedExtensions: ['jpg', 'png', 'gif'],
+              allowMultiple: true))
+          ?.files;
+    } on PlatformException catch (e) {
+      _logException('Unsupported Operation  ${e.toString()}');
+    } catch (e) {
+      _logException(e.toString());
+    }
+    if (!mounted) return;
+    setState(() {
+      _fileName = _paths != null ? _paths!.map((e) => e.name).toString() : null;
+
+      src = _paths != null ? _paths!.map((e) => e.path).toString() : null;
+      imageName = _fileName != null ? _fileName! : null;
+      widget.onSubmitted(src!);
```

Comment:
Adding eg. this class:
```
class FileInfo {
  String? name;
  String? src;
}
```

```suggestion
      if (paths != null) {
        final FileInfo fileInfo = _paths!.map((e) => FileInfo(fileName: e.name, src: e.path));
        
        imageName = fileInfo.name;
        
        if (fileInfo.src != null) {
          widget.onSubmitted(fileInfo.src);
        }
      }
```